packages/react/src/components/Menu/MenuDivider.tsx

```diff
@@ -40,19 +47,19 @@ class MenuDivider extends UIComponent<ReactProps<MenuDividerProps>, any> {
   }
 
   static propTypes = {
-    ...commonPropTypes.createCommon({ content: false, children: false, color: true }),
+    ...commonPropTypes.createCommon({ color: true }),
     primary: PropTypes.bool,
     secondary: PropTypes.bool,
     vertical: PropTypes.bool,
   }
 
   renderComponent({ ElementType, classes, unhandledProps, accessibility }) {
+    const { children, content } = this.props
+
     return (
-      <ElementType
-        {...accessibility.attributes.root}
-        {...unhandledProps}
-        className={classes.root}
-      />
+      <ElementType {...accessibility.attributes.root} {...unhandledProps} className={classes.root}>
```

Comment:
Please add this to the ElementType
```
{...rtlTextContainer.getAttributes({ forElements: [children, content] })}
```
This will ensure that if the children or content are strings, there will be `dir='auto'` applied.